### DISCUSSION TITLE
Can you use standard block editor alongside this theme?

### DISCUSSION THREAD

**Discussion by wpcreative-jw** [flyntwp/flynt#429] [Q&A] [answered]
Can you use standard block editor for certain post types? Things such as blog posts it would be useful to use the block editor for. Thanks

**Comment by aaronmeder** [comment, marked as answer]
Yes definitely. I use either Gutenberg or the classic WP editor one some sites for blog posts.

Inside templates/single.php you could do something like:
```
{# Display the classic WP content field #}
{{ post.content }}
```

If you want to use it for specific post type other than post then make a copy of single.php and name it `single-your_cpt_slug.php`. Then make a copy of templates/single.twig and adjust that to load the classic editor as specified above.

Hope that helps
Aaron
